### PR TITLE
List Structure

### PR DESCRIPTION
**Pull request by joshualyngdoh**
# Description
- Highlights the use of Webpack
- Ran linters check
- Updated Readme

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  tasksList.forEach((task) => {
+    tasks += `<li class="todo-item">
+    <input type="checkbox" name="check" id="check">
+    <input type="text" name="task" id="task" value="${task.description}">
+    <i class="fa-solid fa-trash-can"></i>
+    </li>`;
+  });
+
+  todoList.innerHTML = tasks;
+};
```
Comment (gemmen29) [inline src/index.js]: - [x] Kindly sort the list based on their index value to achieve this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=The%20list%20should%20appear%20in%20order%20of%20the%20index%20values%20for%20each%20task.). The list should appear in order of the index values for each task.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @joshualyngdoh, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (joshualyngdoh) [reply src/index.js]: Thank you for the feedback

Comment (rloterh) [reply src/index.js]: well resolved! ✔️

File: README.md [hunk 2]
```diff
@@ -257,26 +220,12 @@ If you like this project...
...
-
-- **[Question_2]**
-
-  - [Answer_2]
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
```
Comment (rloterh) [inline README.md:231]: - [x] OPTIONAL
MIT license link in your ReadMe is broken, kindly endeavor to fix it, and remove the guide note below it.

File: README.md [hunk 3]
```diff
@@ -99,9 +65,7 @@ After you're finished please remove all the comments and instructions!
 
 > Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **[Highlights the use of Webpack]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (rloterh) [inline README.md:71]: - [x] OPTIONAL
Kindly ensure to remove redundant demo links in your ReadMe.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Josh,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 🎯 
- Project is set up correctly
- Webpack is correctly configured

## Optional suggestions 🧰 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

_Kindly consider my inline comments marked **[OPTIONAL]** under this review_

Cheers..Happy coding! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**